src/Service/EntityMetadata.php

```diff
@@ -361,4 +379,24 @@ protected function findIliosDtos(KernelInterface $kernel): array
 
         return [...$dtos, ...$classes];
     }
+
+    /**
+     * Get an Entity class name for each DTO type
+     */
+    protected function getEntitiesForDtoTypes(): array
+    {
+        $rhett = [];
+        foreach ($this->iliosDtos as $className) {
+            $ref = new ReflectionClass($className);
+            $type = $this->extractType($ref);
+
+            $name = substr($className, 15, -3);
```

Comment:
will need at least a code comment here that we're stripping off the DTO suffix.

Reply:
✔️